R/stabselgraphical.R

```diff
@@ -0,0 +1,67 @@
+#' Create a regularization path - copied from pulsar.
```

Comment:
In general I very much appreciate using roxygen, yet, it is not used in this package so far. Thus I would kindly ask you to remove roxygen and use standard Rd files instead.